app/controllers/CampaignsController.scala

```diff
@@ -16,13 +18,35 @@ class CampaignsController(
   components: ControllerComponents,
   stage: String,
   runtime: zio.Runtime[ZEnv],
-  dynamo: DynamoChannelTests
+  dynamoChannelTests: DynamoChannelTests,
+  dynamoCampaigns: DynamoCampaigns
 )(implicit ec: ExecutionContext)
   extends S3ObjectController[Campaigns](authAction, components, stage, filename = "campaigns.json", runtime) {
 
-  def getTests(campaignName: String) = authAction.async { request =>
+  override def get(): Action[AnyContent] = authAction.async { request =>
```

Comment:
you can also remove `override` here once it's no longer extending `S3ObjectController`